content/lua/concepts/mathematical-library/mathematical-library.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Mathematical Library' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: "Lua's Mathematical Library holds built-in functions relating to math." # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+ - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+The **Mathematical Library** holds Lua built-in functions that relate to math. Just as `print()` is a built-in function, Lua offers other functions for specific use cases.
+
+## Math Functions
+Lua’s Mathematical Library contains various math functionalities, but here are a few most commonly used functions:
+
+#### `math.max(x, ..)`
```

Comment:
These should likely be terms but thats out of scope for the internship. Make note of this in the handoff doc